client/view/index/Index.html

```diff
@@ -1,7 +1,12 @@
 <div class="container">
-	<div class="content">
+
+	<!--Header-->
```

Comment:
No need for all these HTML comments - that's the beauty of using [semantic tags](https://internetingishard.com/html-and-css/semantic-html/), you can tell what's going on by looking at the tag names instead of needing to add something else to distinguish it! 😄